### PR TITLE
Toolkit: Provide update plugin to allow plugins configuration to normalize

### PR DESCRIPTION
**Pull request by srclosson**
Why a new plugin?

This is to allow a plugins configuration to be normalized. Some things that may fall under this category might be:
- Update a circleci config (the purpose of this PR)
- Update package json with a new common library, or update a library
- Add a common parameter to a plugin.json

### CODE REVIEW

**Comment by srclosson** [comment]
Tested against adx. Config updated.

File: packages/grafana-toolkit/src/cli/index.ts [hunk 1]
```diff
@@ -228,6 +229,13 @@ export const run = (includeInternalScripts = false) => {
       });
     });
 
+  program
+    .command('plugin:update')
+    .description('Update plugin')
```
Comment (ryantxu) [inline packages/grafana-toolkit/src/cli/index.ts:234]: update seems too general.... maybe `update-circleci`?

Comment (srclosson) [reply packages/grafana-toolkit/src/cli/index.ts:234]: What about `npx grafana-toolkit plugin:update --circleci` ? I was trying to think about it quite generally.

Comment (srclosson) [reply packages/grafana-toolkit/src/cli/index.ts:234]: Like if you wanted to 'update' a plugin, one of the things might be circleci. But you might have a filter `--filter=circleci`

**Comment by srclosson** [comment]
I'm hoping you can tell me why this test is failing Ryan?

**Review by ryantxu — APPROVED** [review]
LGTM

**Pull request merged**